c++/src/kj/encoding.c++

```diff
@@ -371,9 +371,13 @@ static Maybe<uint> tryFromOctDigit(char c) {
 }  // namespace
 
 String encodeHex(ArrayPtr<const byte> input) {
-  return strArray(KJ_MAP(b, input) {
-    return heapArray<char>({HEX_DIGITS[b/16], HEX_DIGITS[b%16]});
-  }, "");
+  // Do not use KJ_MAP here, we want to do this with a single memory allocation.
```

Comment:
Wow I'm really surprised to see this code was written to do O(n) allocations! It looks like I wrote it, I don't know what I was thinking.

I think we can delete this comment -- there's no reason someone would expect to see KJ_MAP here, so I think the comment is more distracting than useful.